lesson_09/oop/oop_app/src/test/java/com/codedifferently/lesson9/sherllin/BookTest.java

```diff
@@ -0,0 +1,123 @@
+package com.codedifferently.lesson9.sherllin;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+// We want to define a book and keep track of its title, isbn, author(s), number of pages, and
```

Comment:
Don't leave comments in the code unless it's explaining code or documenting a function.